test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -230,6 +260,7 @@ try
 
         Write-LogPassThru -Message "pulling sparse repo"
         "src" | out-file -encoding ascii .git\info\sparse-checkout
```

Comment:
Search replace `out-file` with `Out-File`

Reply:
And casing parameters too.

Reply:
Fixed.